packages/client/src/components/LogoutButton/LogoutButton.tsx

```diff
@@ -6,6 +6,8 @@ import { RootState } from '../../redux';
 import { logout } from '../../redux/User/action';
 import { withRouter, RouteComponentProps } from 'react-router-dom';
 
+// TODO: remove this file if this is not needed for responsive design
```

Comment:
What do you mean by this hahaha. For mobile we show a hamburger menu with all the links right

Reply:
lol some website has dedicated logout button on mobile view 😛, but we are not doing that, let me remove it